src/main/java/mate/academy/spring/controller/ShoppingCartController.java

```diff
@@ -34,14 +35,15 @@ public ShoppingCartController(ShoppingCartService shoppingCartService,
     }
 
     @PutMapping("/movie-sessions")
-    public void addToCart(@RequestParam Long userId, @RequestParam Long movieSessionId) {
+    public void addToCart(Authentication auth, @RequestParam Long movieSessionId) {
         shoppingCartService.addSession(
-                movieSessionService.get(movieSessionId), userService.get(userId));
+                movieSessionService.get(movieSessionId),
+                userService.findByEmail(auth.getName()).get());
     }
 
     @GetMapping("/by-user")
-    public ShoppingCartResponseDto getByUser(@RequestParam Long userId) {
-        User user = userService.get(userId);
+    public ShoppingCartResponseDto getByUser(Authentication auth) {
+        User user = userService.findByEmail(auth.getName()).get();
```

Comment:
Please use .orElseThrows() instead of get and provide proper exception message inside.
Please recheck whole code, there are several occurrences